packages/simulator/web/routes.js

```diff
@@ -116,22 +115,26 @@ function register(io) {
 
           const averageUtilization = cars.pipe(
             mergeMap(car => fromEvent(car, 'cargo')),
-            windowTime(15000),
-            map(window => window.pipe(
-              distinct(car => car.id),
```

Comment:
@irony turns out that distinct gives only the latest value per key. In the following example id 1 has value 2, id 2 has value 0

```es6
const { of, from, timer } = Rx;
const { distinct, map, delay, concatMap } = RxOperators;

const cars$ = from([
  {id: 1, value: 2},
  {id: 1, value: 1337},
  {id: 2, value: 0},
  {id: 1, value: 1447},
  {id: 2, value: 10},
])

cars$.pipe(
  concatMap(x => of(x).pipe(delay(1000))),
  distinct(c => c.id),
)
```